tests/test_autonames/test_autonames.py

```diff
@@ -10,11 +10,20 @@
 def test_autonames():
     """Here we check that bad migrations do produce warnings."""
     warnings = check_migration_names()
+    warnings_msgs = {warning.msg for warning in warnings}
 
-    assert len(warnings) == 1
-    assert warnings[0].level == WARNING
-    assert warnings[0].id.startswith(CHECK_NAME)
-    assert 'main_app.0004_auto_20191119_2125' in warnings[0].msg
+    assert len(warnings) == 2
+
+    assert [warnings[0].level, warnings[1].level] == [WARNING, WARNING]
```

Comment:
I wrote so In order to pass the violation check – WS218. But, I think, in this case, **subtest** (https://github.com/pytest-dev/pytest-subtests) would be more convenient.